PACKAGES.md

```diff
@@ -0,0 +1,103 @@
+# Packaging Frescobaldi
+
+## Versioning and release best practices
+
+Release versioning should follow [PEP 440](https://peps.python.org/pep-0440/).
+
+The version number should be bumped right after a release and have a
+`.dev1` suffix to mark it as a developmental release.
+For example, after 4.0.0 release the version will be bumped to
+4.0.1.dev1 so any package built during the development will have
+a version different from the actual releases. Right before the
+release the `.dev1` suffix can be removed.
+
+Before creating a new release on GitHub make sure to review this
+**release checklist**:
+
+- Update the version in `frescobaldi/appinfo.py` and `pyproject.toml`.
+- Check that project.dependencies and tool.briefcase.requires versions
+  in `pyproject.toml` are correct and up-to-date.
+- Make sure that the `CHANGELOG.md` has a complete list of features
+  and bug fixes for the new release. Set the release date and add
+  a link to the release list of commits.
+- Add a new release entry in `linux/org.frescobaldi.Frescobaldi.metainfo.xml.in`.
+- Check if the list of contributors in `frescobaldi/userguide/credits.md`
+  should be updated.
+- Create a source tarball (see the next section).
+
+You can manually run the GitHub Action release workflow on the master
+branch just before making a new release in order to test the packages.
+
+
+## Tarball for Linux distros
+
+The tarball automatically created by GitHub for every release
+is missing some files generated by tox: the .mo files containing
+the translations for all platforms; the desktop and metainfo
+files used for Linux packaging.
+
+The maintainers of Frescobaldi packages for Linux distros don't want
+to run build commands to generate the missing files, so we have to
+provide a tarball with all the needed files already available.
+Here's how to build a compliant tarball using a new clean directory:
+
+```
+git clone -b v4.0.0 git@github.com:frescobaldi/frescobaldi.git frescobaldi-4.0.0
+cd frescobaldi-4.0.0
+tox -e mo-generate
+tox -e linux-generate
+cd ..
+tar --exclude='.git*' --exclude='.tox' --exclude='i18n/__pycache__' -czvf frescobaldi-4.0.0.tar.gz frescobaldi-4.0.0
```

Comment:
Umm, are you really doing this with `tar` manually? Normally this should be done with the build backend, i.e., `python -m build`…

Reply:
Especially since sdists are supposed to contain package metadata, this will not build them in a compliant way…

Reply:
I had the feeling that I'd better use `python -m build`... I will update the command keeping in mind your PR to use hatchling as build backend.

Reply:
I've tested `python -m build` using setuptools and the result is not Ok. Only frescobaldi directory and a few files in the top dir are copied.
The same command using hatchling seems perfect.